### PR TITLE
Feature/sdl 0188 interior vehicle data resumption

### PR DESCRIPTION
**Pull request by LuxoftAKutsan**
Fixes [1480](https://github.com/smartdevicelink/sdl_core/issues/2661)

:warning:  PR depends on https://github.com/smartdevicelink/sdl_core_guides/pull/94 

This PR is  **ready** for review.

### Risk
N/A
### Testing Plan
N/A

### Summary
Add interior vehicle data to the resumption 

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+note over ResumptionDataProcessor
+  Responsible 
+  for sending apropriate
+  RAI response when 
+  resumption will be finished
+end note
+
+note over ExtensionPendingResumptionHandler
+  Responsible for watching if subscription
+  to any data already sent
```
Comment (LuxoftAKutsan) [inline docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml:32]: sent to HMI

File: docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  and prevent sending dublicate 
+end note
+
+RAI1 -> ResumptionDataProcessor : StartResumption
```
Comment (LuxoftAKutsan) [inline docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml:36]: remove RAI. reaple with RAI1 or RAI2

File: docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+@startuml
```
Comment (LuxoftAKutsan) [inline docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml:1]: Show explicit that is app2 is subsribed not another data, appropriate HMI request will be sent

Comment (LuxoftAKutsan) [reply docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml:1]: Fixed in   https://github.com/smartdevicelink/sdl_core_guides/pull/94/files

File: docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml (hunk 1 above)
Comment (LuxoftAKutsan) [reply docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml]: Fixed in   https://github.com/smartdevicelink/sdl_core_guides/pull/94/files

File: docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml (hunk 2 above)
Comment (LuxoftAKutsan) [reply docs/Developer Documentation/ResumeController/assets/subscriptoins_restore_sequence.puml:37]: Fixed in   https://github.com/smartdevicelink/sdl_core_guides/pull/94/files

**Comment by KhrystynaDubovyk** [comment]
@yang1070 please, see the updates made

**Comment by KhrystynaDubovyk** [comment]
@ShobhitAd, the conflicts were resolved. The PR is ready for merging.

**Review by ShobhitAd — APPROVED** [review]

**Pull request merged**